features/cli/add_account.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Storing AWS accounts
+
+  In order to manage AWS accounts
+  As an AWS user
+  I want to store an AWS account
+  So I can reference an account by name instead of by ID
+
+  Scenario: Adding an AWS account
```

Comment:
Potentially retrieve the [IAM Account Alias](http://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccountAliases.html) if the user is already authenticated, and the iam:ListAccountAliases action is permitted.